releasetasks/templates/final_verify.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+{% set buildername = "{}_final_verify".format(branch) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: b2gtest
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+
+        payload:
+            maxRunTime: 7200
+            image: rail/python-test-runner
```

Comment:
I'm assuming that you will be doing that (You made a comment before that you had to) so also assuming non-blocking